pipelines/zap/pipeline.yml

```diff
@@ -88,9 +88,9 @@ jobs:
       put: slack
       params:
         channel: <%= project['slack_channel'] ? "'##{project['slack_channel']}'" : "{{slack-channel}}" %>
-        icon_url: {{slack-icon}}
```

Comment:
I didn't really mean to push this part. It does work though. Is an emoji better than an icon_url?  If so I'll just adjust the {{config}}